package.json

```diff
@@ -50,6 +51,9 @@
       "name": "Google Inc."
     }
   ],
+  "engines": {
+    "node": "^8.13.0 || >=10.10.0"
```

Comment:
I don't really see a point to this, because this package isn't published and this range isn't binding anyway.

Reply:
It's advisory the same way it is in `grpc-js`, I've removed it.